baseframe/forms/fields.py

```diff
@@ -10,14 +10,16 @@
 from wtforms.widgets import Select as OriginalSelectWidget
 from wtforms.compat import text_type
 from wtforms.utils import unset_value
+from wtforms.validators import StopValidation
 import bleach
+import simplejson as json
 import six
 
 from .. import _, get_timezone
 from .widgets import TinyMce3, TinyMce4, DateTimeInput, CoordinatesInput, RadioMatrixInput, SelectWidget, Select2Widget
 from .parsleyjs import TextAreaField, StringField, URLField
 
-__all__ = ['SANITIZE_TAGS', 'SANITIZE_ATTRIBUTES',
+__all__ = ['SANITIZE_TAGS', 'SANITIZE_ATTRIBUTES', 'JsonField',
```

Comment:
The `__all__` list needs to be sorted according to one linter. Could you do that?